spec/spec_helper.rb

```diff
@@ -30,6 +32,10 @@
     ex.run_with_retry retry: 3
   end
 
+  config.before :suite do |_ex|
+    WebMock.disable!
```

Comment:
Isn't this going to disable WebMock for all the tests? We don't want that, right?